src/gui/qgsmaptoolcapture.h

```diff
@@ -137,6 +132,12 @@ class GUI_EXPORT QgsMapToolCapture : public QgsMapToolAdvancedDigitizing
      */
     void closePolygon();
 
+  protected slots:
+    /**
+     * Stop capturing
+     */
+    void stopCapturing();
```

Comment:
Please add a

    @note TODO QGIS 3: make protected

instead, this may break some things